interface/src/app/gene/network/network.js

```diff
@@ -235,16 +255,29 @@ angular.module('adage.gene.network', [
           .call(edgeTip);
 
         geneTip.html(getGeneInfo);
-        network.onGene('mouseover.custom', geneTip.show);
-        network.onGene('mouseout.custom', geneTip.hide);
-        network.onEdge('mouseover.custom', showEdgeTip);
-        network.onEdge('mouseout.custom', hideEdgeTip);
+        network.onGene('click.custom', geneTip.show);
+        network.onEdge('click.custom', showEdgeTip);
```

Comment:
It sounds as if you've discussed the design of this interaction with @cgreene directly, so I won't second-guess the rationale for this change, but I do want to add my 2¢ here. The nice thing about responding to a `mouseover` event in some way is that the feature is easy to discover. It's not necessarily obvious that clicking will do anything. In fact, I had to read the comments on this PR and in the code a bit to realize the link feature was being added to the tip that comes up when clicking on an *edge*. Not sure I would have thought to click while hovering over an edge otherwise.

If you've done user testing that indicates that this is not a problem, then all is good. I anticipate that this feature will be hard to discover for a new user, though.